pkgs/development/compilers/pakcs/default.nix

```
@@ -4,13 +4,13 @@
 , curl, git, unzip, gnutar, coreutils, sqlite }:
 
 let
-  name = "pakcs-2.1.1";
+  name = "pakcs-2.1.2";
 
   # Don't switch to development release without a reason, because its
   # source updates without version bump. Prefer current release instead.
   src = fetchurl {
     url = "https://www.informatik.uni-kiel.de/~pakcs/download/${name}-src.tar.gz";
-    sha256 = "112v9ynqfbbm4x770mcfrai9v5bh7c3zn7jka80pv6v4y65r778c";
+    sha256 = "00irjlyfh1888bnvrjg477jkfcxhd76ysh3jr13r3bx80bkih0cj";
```

i got a d different hash
```suggestion
    sha256 = "0i0nprli3knc7zlp5qkqkpiq3ny36v52hnvgph376l3ajjds7wf6";
```
```
warning: ignoring the user-specified setting 'sandbox', because it is a restricted setting and you are not a trusted user
hash mismatch in fixed-output derivation '/nix/store/8s2d859g5r8d6gchcndn9alaqlf44vgy-pakcs-2.1.2-src.tar.gz':
  wanted: sha256:00irjlyfh1888bnvrjg477jkfcxhd76ysh3jr13r3bx80bkih0cj
  got:    sha256:0i0nprli3knc7zlp5qkqkpiq3ny36v52hnvgph376l3ajjds7wf6
```

---

after, changing the hash I still got:

```
NEW COMPILED LIBRARIES IN THIS ITERATION:
scripts/compile-all-libs.sh: line 24: /bin/rm: not found
```
looks like you may need to patch some commands :(